cmd/zetaclientd/debug.go

```diff
@@ -97,10 +97,10 @@ func DebugCmd() *cobra.Command {
 
 			if chains.IsEVMChain(chain.ChainId) {
 
-				ob := evm.ChainClient{
+				ob := evm.Client{
```

Comment:
```suggestion
				obEVM := evm.Client{
```

Reply:
I replaced `EVM Client` and `BTC Client` with the new name `Observer` in https://github.com/zeta-chain/node/pull/2150/commits/6a2a68a8c776e333d5bad07894ab4962fba64d17 . A new `Client` struct that contains both `Observer` and `Signer` will be introduced in coming PR.